client/components/payment-method-logos/style.scss

```
@@ -0,0 +1,30 @@
+.woocommerce-payments-method-logos_count {
+	width: 38px;
+	height: 24px;
+	background-color: #f2f2f2;
+	text-align: center;
+	line-height: 24px;
+	border-radius: 3px;
+	font-size: 11px;
+	font-weight: 600;
+}
+
+.woocommerce-payments-method-logos > svg {
+	width: 38px;
+	height: 24px;
+}
+
+.woocommerce-payments-method-logos img {
+	margin-right: 8px;
+}
+
+.connect-account-page__payment-methods .woocommerce-payments-method-logos {
+	border: 1px solid #ddd;
+	border-bottom: none;
+	padding: 12px;
+}
+
+.woocommerce-payments-method-logos {
+	display: flex;
+	align-items: center;
+}
```

```suggestion
.connect-account-page__payment-methods--logos {
	display: flex;
	align-items: center;
	border: 1px solid $gray-300;
	border-bottom: none;
	padding: 12px;

	img {
		margin-right: 8px;
	}
	
	&-count {
		width: 38px;
		height: 24px;
		background-color: #f2f2f2;
		text-align: center;
		line-height: 24px;
		border-radius: 3px;
		font-size: 11px;
		font-weight: 600;
	}
}
```

I recommend using the SCSS structure.

---

fixed in [6564ac2](https://github.com/Automattic/woocommerce-payments/pull/8977/commits/6564ac2b72c939d990a7ad759fb88aa2d8cef9d0) . thx